### ISSUE TITLE
How to disable zoom in/out?

### ISSUE THREAD

**Issue by ntnamag** [iddan/react-native-canvas#17]
I still can zoom canvas, how to turn it off? If adding meta tags to html is no longer standard when updating the component

**Comment by iddan** [comment]
We should not support zoom. Which tags do you use to disable it?

**Comment by ntnamag** [comment]
Add to html file meta tag work fine
`<head>
 <meta content='width=device-width, initial-scale=1, maximum-scale=1, user-scaleable=no' name='viewport'>
</head>`

**Comment by iddan** [comment]
text?

**Comment by ntnamag** [comment]
```
<html>
<head>
 <meta content='width=device-width, initial-scale=1, maximum-scale=1, user-scaleable=no' name='viewport'>
</head>
  <body>
    <script src="./webview.js"></script>
  </body>
</html>

```

**Comment by iddan** [comment]
Fixed in #19
